File: RoomLiveDataSampleJava/app/src/main/java/com/websarva/wings/android/roomlivedatasamplejava/data/local/Cocktailmemo.java
package com.websarva.wings.android.roomlivedatasamplejava.data.local;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "cocktailmemos")
public class Cocktailmemo {
	@PrimaryKey(autoGenerate = true)
	public long id;
	@NonNull
	public String name;
	@NonNull
	public int price;
}

File: RoomFlowSample/app/src/main/java/com/websarva/wings/android/roomflowsample/data/local/Cocktailmemo.kt
package com.websarva.wings.android.roomflowsample.data.local

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "cocktailmemos")
data class Cocktailmemo (
	@PrimaryKey(autoGenerate = true) val id: Long,
	val name: String,
	val price: Int
)
